sources/mongodb/helpers.py

```diff
@@ -37,40 +37,62 @@ def __init__(
         self.client = client
         self.collection = collection
         self.incremental = incremental
+
         if incremental:
             self.cursor_field = incremental.cursor_path
-            self.last_value = incremental.last_value
         else:
-            self.cursor_column = None
```

Comment:
That seems like a mistake: `cursor_field` and `cursor_column`, while the second one is never used